specs/simple-serialize.md

```diff
@@ -78,8 +78,18 @@ For convenience we alias:
 * `null`: `{}`
 
 ### Default values
-
-The default value of a type upon initialization is recursively defined using `0` for `uintN`, `False` for `boolean` and the elements of `Bitvector`, and `[]` for lists and `Bitlist`. Unions default to the first type in the union (with type index zero), which is `null` if present in the union.
+Assuming a helper function `default(type)` which returns the default value for `type`, we can recursively define the default value for all types.
+
+| Type | Default Value
+| ---- | ------------- |
+| `uintN` | `0` |
+| `boolean` | `False` |
+| `Vector[type, N]` | `[default(type)] * N]` |
+| `Bitvector[boolean, N]` | `[False] * N` |
+| `List[type, N]` | `[]` |
+| `Bitlist[boolean, N]` | `[]` |
+| `Union[type_0, type_1, ...]` | `default(type_0)` |
+| `Container` | `[default(type) for type in container]` |
```

Comment:
nitpick: order by [how they were introduced in this spec](https://github.com/ethereum/eth2.0-specs/blob/67a893b585c3bc6907d37108dc2bf45e6f9e8f3a/specs/simple-serialize.md#typing).

```
| ---- | ------------- |
| `uintN` | `0` |
| `boolean` | `False` |
| `Container` | `[default(type) for type in container]` |
| `Vector[type, N]` | `[default(type)] * N]` |
| `List[type, N]` | `[]` |
| `Bitvector[boolean, N]` | `[False] * N` |
| `Bitlist[boolean, N]` | `[]` |
| `Union[type_0, type_1, ...]` | `default(type_0)` |
```